<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseConversion</class>
 <widget class="QWidget" name="CaseConversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>527</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>90</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_upper">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 4px;
    background-color: #66afe9;
    border: none;
    color: #FFFFFF;
    text-align: center;
    font: 11pt &quot;微软雅黑&quot;;
    margin: 2px;
    vertical-align: middle;
}

QPushButton:hover {
    opacity: 1;
    right: 0;
    background-color: #1976d2;
}
</string>
   </property>
   <property name="text">
    <string>全大写</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lower">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>40</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 4px;
    background-color: #66afe9;
    border: none;
    color: #FFFFFF;
    text-align: center;
    font: 11pt &quot;微软雅黑&quot;;
    margin: 2px;
    vertical-align: middle;
}

QPushButton:hover {
    opacity: 1;
    right: 0;
    background-color: #1976d2;
}
</string>
   </property>
   <property name="text">
    <string>全小写</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_left">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>591</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>170</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
font-size: 14px;
color: #000;
font: 11pt &quot;微软雅黑&quot;;
background-color: #ffffff;
border: 1px solid #cccccc;
border-radius: 4px;
}

QPlainTextEdit:focus {
border-color: #66afe9;
outline: 0;
}</string>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::WidgetWidth</enum>
   </property>
   <property name="placeholderText">
    <string>Input:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_right">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>591</width>
     <height>170</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>170</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>250</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
font-size: 14px;
color: #000;
font: 11pt &quot;微软雅黑&quot;;
background-color: #ffffff;
border: 1px solid #cccccc;
border-radius: 4px;
}

QPlainTextEdit:focus {
border-color: #66afe9;
outline: 0;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Output:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_upper</tabstop>
  <tabstop>btn_lower</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
